isp/network/interfaces.d/dummy0

```diff
@@ -0,0 +1,3 @@
+auto dummy0 
+iface dummy0 inet static
+  address 172.23.44.130/32
```

Comment:
Use 172.23.44.130 which is in IP range for external services, this IP address is the authoritative server.